packages/rest/test/acceptance/bootstrapping/rest.acceptance.ts

```diff
@@ -27,7 +27,7 @@ describe('Bootstrapping with RestComponent', () => {
       class UserDefinedSequence extends DefaultSequence {}
       app = new Application({
         components: [RestComponent],
-        rest: {
+        RestServer: {
```

Comment:
Do we want to qualify these keys under `servers`, for example:
```
{
  components: [RestComponent],
  servers: {
    RestServer: {
    }
  }
}
```

Reply:
We can't, since that's how people will introduce server constructors without components.